content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,196 @@
+that is descriptive of the channel's purpose, and has no spaces or commas.
+The name you choose will become what's called your namespace. You can use
+some punctuation such as `.`, `_` or `+` instead of spaces, but it is better
+to just drop the space. It is typical to reserve `-` as a delimiter for
+additional channels within the Namespace, such as `-dev` or `-ops`.
 
-The next step of creating a channel is to `/join` it.
+Once you're settled on the name, add the appropriate number of `#`'s to the
+beginning, according to the rules in [our namespaces policy](/chanreg).
+
+For example, if it is an official representation of your project and the
+project is named "Awesome App", you can use one `#`. You'll probably go with
+something like: `#AwesomeApp`
+
+If it's not a "Project" or "Community" channel based on the definitions
+described in [our namespaces policy](/chanreg) you'd go with two `#`. As an
+example, it might be a channel for your friends to talk to you in, so you'd
+probably pick something like `##YourNickname`
+
+## Check that it is available
+
+You can check that a channel is registered or not with `ChanServ`. To see if
+for example `#libera` is registered, you would do:
+
+```irc
+/msg ChanServ info #libera
+```
+
+Since it is registered, ChanServ will tell you the registration details you're
+allowed to see. Otherwise it would say:
+
+```irc
+ChanServ: #libera is not registered.
+```
 
-Pick a channel name without spaces and add the appropriate number of `#`'s to
-the beginning, according to [our namespaces policy](/chanreg).
-Eg, `/join #awesomeprojectname` if it is for an official project channel
-or a community channel about a project, or `/join ##yournickname-chats` for an
-informal channel.
+## Joining the channel
 
-### Check that it is available
+The next step of creating a channel is to `/join` it.
+
+```irc
+/join <#channelname>
+```
 
-You can check that a channel is registered or not with
-`/msg ChanServ info #wanted-channel-name`
+## Registering with `ChanServ`
 
-If it is not registered ChanServ will tell you, otherwise it will show you
-the public information for the existing channel.
+If the channel you joined was empty and is unregistered, you will have the
+temporary Operator status now. Depending on your client, you should appear in
+the user list with an `@` or another symbol before your nick.
 
-The first person to join an unregistered channel is granted temporary operator
-status. If the channel is empty and unregistered, you should have operator
-status now. If not, you will need to talk to the people already in the
-channel, or if there is nobody there with Libera.Chat staff either in
-`#libera` or on `/stats p`.
+If you do not get the Operator status it could be because there are people
+already in the channel. You will need to ask them to leave while you set it
+up. If they do not cooperate, or if the channel is in fact already registered,
```

Comment:
In a lot of cases if the existing members don't cooperate the solution would be to pick a new name rather than to talk to staff. We'd only need to be consulted for an on-topic registration for a namespace assignment or something like that.

Reply:
aaand i'm rewriting this bit again now and it's all your fault